get.go

```diff
@@ -172,33 +173,46 @@ func (j *JWKs) refresh() (err error) {
 		return err
 	}
 
-	// Get the JWKs as JSON from the given URL.
+	// Get the JWKS as JSON from the given URL.
 	var resp *http.Response
 	if resp, err = j.client.Do(req); err != nil {
 		return err
 	}
 	defer resp.Body.Close() // Ignore any error.
 
-	// Read the raw JWKs from the body of the response.
+	// Read the raw JWKS from the body of the response.
 	var jwksBytes []byte
 	if jwksBytes, err = ioutil.ReadAll(resp.Body); err != nil {
 		return err
 	}
 
-	// Create an updated JWKs.
-	var updated *JWKs
+	// Checksum the raw JWKS.
+	h := fnv.New64a()
+	if _, err = h.Write(jwksBytes); err != nil {
+		return err
+	}
+	checksum := h.Sum64()
+
+	// Reprocess the JWKS if the checksum doesn't match.
+	if checksum == j.fnv {
+		return nil
+	}
+	j.fnv = checksum
```

Comment:
Just keep the previous jwksBytes, it is simpler and faster to compare to.

Reply:
Yeah this makes sense. I did a quick benchmark, it's also about 50x faster. I don't think anyone would complain about a couple extra bytes of memory either.

Added in this commit: https://github.com/MicahParks/keyfunc/pull/25/commits/1c9ade36ad129fba923b141f26bd59137c2ffff2